core/workspace_svg.js

```diff
@@ -2486,6 +2492,68 @@ Blockly.WorkspaceSvg.prototype.getTopBlocks = function(ordered) {
   return Blockly.WorkspaceSvg.superClass_.getTopBlocks.call(this, ordered);
 };
 
+/**
+ * Add a block to the list of top blocks.
```

Comment:
Adds
(based on [style guide](https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments) "written in the third person, as if there is an implied This method ... before it.")

Similar comment for following function JSDocs.

Reply:
Done, and fixed parent methods.